code/web/src/modules/user/EditProfileForm.js

```diff
@@ -0,0 +1,193 @@
+                placeholder="Name"
+                required="required"
+                name="name"
+                value={this.state.user.name}
+                onChange={this.onChange}
+              />
+
+              {/* Email */}
+              <Input
+                type="email"
+                fullWidth={true}
+                placeholder="Email"
+                required="required"
+                name="email"
+                value={this.state.user.email}
+                onChange={this.onChange}
+                style={{ marginTop: '1em' }}
+              />
+
+              {/* Password */}
+              <Input
+                type="password"
+                fullWidth={true}
+                placeholder="Password"
+                required="required"
+                name="password"
+                value={this.state.user.password}
+                onChange={this.onChange}
+                style={{ marginTop: '1em' }}
+              />
+
+              {/* Image */}
+              <Input
+                type="image"
+                fullWidth={true}
+                placeholder="Image"
+                name="image"
+                value={this.state.user.image}
+                onChange={this.onChange}
+                style={{ marginTop: '1em' }}
+              />
+
+              {/* Description */}
+              <Input
+                type="text"
+                fullWidth={true}
+                placeholder="Description"
+                name="description"
+                value={this.state.user.description}
+                onChange={this.onChange}
+                style={{ marginTop: '1em' }}
+              />
+
+              {/* Shipping Address */}
+              <Input
+                type="text"
+                fullWidth={true}
+                placeholder="Shipping Address"
+                required="required"
+                name="shipping"
+                value={this.state.user.shipping}
+                onChange={this.onChange}
+                style={{ marginTop: '1em' }}
+              />
+
+              {/* Available Date ??? */}
+              <Input
+                type="date"
+                fullWidth={true}
+                placeholder="Available Date"
+                name="availableDate"
+                value={this.state.user.availableDate}
+                onChange={this.onChange}
+                style={{ marginTop: '1em' }}
+              />
+            </div>
+
+            <div style={{ marginTop: '2em' }}>
+              {/* Profile Form link */}
+              <Link to={userRoutes.editProfile.path}>
+                <Button type="button" style={{ marginRight: '0.5em' }}>Submit</Button>
+              </Link>
+
+              {/* Form submit */}
+            </div>
+          </form>
+
+      </Grid>
+    )
+  }
+}
+
+
+
+function profileState(state) {
+  return {
+    user: state.user
+  }
+}
+
+export default connect(profileState, { logout })(EditProfileForm)
```

Comment:
Great job building out the component a creating the route! Recycling the UI components seems like a huge help in staying with code consistency. We will have to do some tweaks, but this is fantastic start to our Edit Profile form